tests/test_printer.py

```diff
@@ -870,12 +870,14 @@ def test_get_printed_name():
     val = TestSSAValue(i32)
 
     # Test printing without constraints
-    printer.stream = StringIO()
+    stream = StringIO()
+    printer.stream = stream
```

Comment:
Why?!

Reply:
because the instance on the printer is nullable, whereas the local variable is not